smdebug/core/hook.py

```diff
@@ -1,6 +1,8 @@
 # Standard Library
 import atexit
+import os
 import re as _re
+import socket
```

Comment:
needed for?

Reply:
Used in the line

```
self.logger.info(
                        f"Hook is writing from the hook with pid: {os.getpid()}\n"
                        f"and hostname: {socket.gethostname()}"
                    )
```

Reply:
Please check in SM script that you can use socket.gethostname and it doesn't error out. 
It can be any script running in SM doing  - 
import socket
print(socket.gethostname())